crates/sui-indexer/migrations/pg/2024-09-27-233849_add_first_tx_sequence_number_to_epochs/up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE epochs
+ADD COLUMN first_tx_sequence_number BIGINT;
```

Comment:
Will the reader be reading from this column as well?

Reply:
This is p much just for the pruner, the `lo` and `hi` for an entry will be in the unit expected of the entry. For `transactions`, it's `lo` and `hi` will be tx seq num